frontend/assignments/3/frontend/src/components/stockpage/StockPage.tsx

```
@@ -0,0 +1,452 @@
+                    >
+                      {price &&
+                      price < priceHistory[priceHistory.length - 2] ? (
+                        <ArrowDownward />
+                      ) : (
+                        <ArrowUpwardIcon />
+                      )}
+                    </div>
+                    <div></div>
+                  </div>
+                  {price &&
+                  !isNaN(
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)